hw5/hw5_t01.py

```diff
@@ -0,0 +1,81 @@
+"""
+Необходимо создать 3 класса и взаимосвязь между ними.
+
+(Student, Teacher, Homework)
+Наследование в этой задаче использовать не нужно.
+Для работы с временем использовать модуль datetime
+
+1. Homework принимает на вход 2 атрибута: текст задания и количество дней
+на это задание
+Атрибуты:
+    text - текст задания
+    deadline - хранит объект datetime.timedelta с количеством
+    дней на выполнение
+    created - c точной датой и временем создания
+Методы:
+    is_active - проверяет не истело ли время на выполнение задания,
+    возвращает boolean
+
+2. Student
+Атрибуты:
+    last_name
+    first_name
+Методы:
+    do_homework - принимает объект Homework и возвращает его же,
+    если задание уже просрочено, то печатет 'You are late' и возвращает None
+
+3. Teacher
+Атрибуты:
+     last_name
+     first_name
+Методы:
+    create_homework - текст задания и количество дней на это задание,
+    возвращает экземпляр Homework
+    Обратите внимание, что для работы этого метода не требуется сам объект.
+
+PEP8 соблюдать строго.
+Всем перечисленным выше атрибутам и методам классов сохранить названия.
+К названием остальных переменных, классов и тд. подходить ответственно -
+давать логичные подходящие имена.
+"""
+import datetime
+from typing import Any
+
+
+class Homework:
+    """Homework class."""
+
+    def __init__(self, text: str, deadline: int):
```

Comment:
Why deadline is int? 

This name does not contain information on how to measure it. Is it second, or hours, or maybe milliseconds?

Reply:
We are accepting an integer number of days and then we store it as `datetime.timedelta`.

Reply:
> This name does not contain information on how to measure it. 

Can we solve it?